<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>765</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>765</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aleks AV</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="gui_tabMain">
     <attribute name="title">
      <string>Scanning</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>741</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="gui_currentScan">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Progress:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="gui_ThreatTable"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="gui_startScan">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start Scan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gui_deleteThreat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gui_quarantineThreat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Quarantine</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gui_tabSettings">
     <attribute name="title">
      <string>Scan Settings</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="gui_searchDir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>591</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Search Directory:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="gui_bMemScan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Running Process Scan</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="gui_bDirScan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="gui_ads">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>511</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gui_tabStatistics">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QChartView" name="gui_scanAverage"/>
       </item>
       <item>
        <widget class="QChartView" name="gui_scanDangerousAverage"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gui_quarantineList">
     <attribute name="title">
      <string>Quarantine List</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableView" name="gui_QuarantineTable"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="gui_restoreThreat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Restore</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gui_quarantineDeleteThreat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount_Details">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionEdit_Details"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSpawnConsole"/>
    <addaction name="actionVersion_Information"/>
   </widget>
   <addaction name="menuAccount_Details"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionEdit_Details">
   <property name="text">
    <string>View Details</string>
   </property>
  </action>
  <action name="actionSpawnConsole">
   <property name="text">
    <string>Open Debug Console</string>
   </property>
  </action>
  <action name="actionVersion_Information">
   <property name="text">
    <string>Version Information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
